src/modules/material-requests/components/MaterialRequestDetailBlade.tsx

```diff
@@ -0,0 +1,152 @@
+import { Button } from '@meemoo/react-components';
+import Image from 'next/image';
+import { FC } from 'react';
+
+import { Blade, Icon, IconNamesLight } from '@shared/components';
+import useTranslation from '@shared/hooks/use-translation/use-translation';
+import { formatMediumDate } from '@shared/utils';
+
+import { MaterialRequestDetails } from '@material-requests/types';
+
+interface MaterialRequestDetailBladeProps {
+	isOpen: boolean;
+	onClose: () => void;
+	currentMaterialRequestDetails?: MaterialRequestDetails;
+}
+
+const MaterialRequestDetailBlade: FC<MaterialRequestDetailBladeProps> = ({
+	isOpen,
+	onClose,
+	currentMaterialRequestDetails,
+}) => {
+	const { tText } = useTranslation();
+
+	const renderFooter = () => {
+		return (
+			<div className="p-account-my-material-requests__close-button-container">
+				<Button
+					label={tText('modules/cp/const/material-requests___sluit')}
+					variants={['block', 'text']}
+					onClick={onClose}
+					className="p-account-my-material-requests__close-button"
+				/>
+			</div>
+		);
+	};
+
+	return (
+		<Blade
+			isOpen={isOpen}
+			renderTitle={() => (
+				<h2 className="p-account-my-material-requests__title">
+					{tText('modules/cp/const/material-requests___detail')}
```

Comment:
Translations for this file should start with: `modules/cp/components/material-request-detail-blade/`